ethminer/main.cpp

```diff
@@ -222,7 +222,7 @@ class MinerCLI
         app.add_option("--work-timeout", m_worktimeout,
                "Set disconnect timeout in seconds of working on the same job", true)
```

Comment:
you changed that to milliseconds right?

Reply:
Yes ... in the client is handled as ms.

Reply:
So the description for the command should be changed as-well, sorry could have wrote it clearer in the first place ^^

Reply:
Hmm ... the description and the CLI values are correct to be in seconds (who would ever want to set a work-timeout to 180 ms ?)
On the other hand, internally, all timeouts are handled in milliseconds so m_worktimeout * 1000